vignettes/BERT_basics.Rmd

```diff
@@ -0,0 +1,165 @@
+---
+title: "BERT Basics"
+output: 
+  rmarkdown::html_vignette:
+    toc: true
+vignette: >
+  %\VignetteIndexEntry{BERT Basics}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+ <!-- 
+ Copyright 2019 Bedford Freeman & Worth Pub Grp LLC DBA Macmillan Learning. 
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ -->
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+
+
+
+If you are not familiar with BERT, I suggest you check out the helpful blog post
+[here](http://jalammar.github.io/illustrated-bert/), and the resources linked
+from it.
+
+Until you have time to do that, this vignette is intended to be a quick, bare-bones
+introduction to BERT--just enough so that the rest of this package makes sense.
+
+## What is BERT?
+
+For now, think of BERT as a function that takes in text and puts out numbers--a
+long list of numbers for *each* token (~word) of the input.
+
+In fact, there is really a family of such functions. Google research released
+the first BERT models in late 2018, and others have followed (as well as many
+that include slight modifications to the original structure). When referring to
```

Comment:
Consider breaking at . (and maybe other logical breaks, like ()) to make track changes easier (and not hard-wrapping otherwise).

Reply:
I think I got everything (though I didn't do breaks at () yet).